lib/beaker/host/windows.rb

```diff
@@ -29,7 +29,7 @@ def self.pe_defaults
         'sitemoduledir' => 'C:/usr/share/puppet/modules',
         #let's just add both potential bin dirs to the path
         'puppetbindir'  => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet Enterprise/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet Enterprise/bin',
-        'pathseparator' => ':',
+        'pathseparator' => ';',
```

Comment:
You might also want to add a note about which commit introduced this errant change (and why it's being undone in this commit)